converter/manifest_to_project.py

```diff
@@ -0,0 +1,160 @@
+import glob
```

Comment:
This PR is just a copy-paste of `converter/dbt_metrics_to_semantic_model_converter.py` & `converter/spec_upgrade.py`. I intend to come back later and clean up this code some more.